app.js

```diff
@@ -0,0 +1,12 @@
+const menBtn = document.querySelector('.nae');
+const navstuff = document.querySelector('.mobile-menu');
+const Closer = document.querySelector('.close-button');
+const rep = document.getElementById('hide');
+
+menBtn.addEventListener('click', () => {
+  menBtn.classList.toggle('navblur');
+  navstuff.classList.toggle('activate');
+  navstuff.classList.toggle('mobile-menu');
+  Closer.classList.toggle('visible');
```

Comment:
Only when a user clicks the X button, the mobile menu should disappear. At the moment, the mobile menu disappears when the user clicks anywhere on the screen.

- [ ] In order to have a nice functional app, kindly, refactor the code in a way that the mobile menu disappears only when the user clicks the "X" button. :100:

Reply:
@nicupop729 Hello sorry are u saying everything is wrong or just the closing button

Reply:
Hello @amklass11!

At the moment, when the user clicks anywhere, the menu disappears.
It should only be when the user clicks the X button.

It is clear?  Please feel free to ask if not. We can talk more on slack. search me there at @Nicoalae Pop

Reply:
@nicupop729 Corrections made what do u think

Reply:
If everything is functional, if the requirements were respected and if you feel prepared, please ask for another review. :smile_cat:

Reply:
Alright thank you